engine/apps/api/views/user.py

```diff
@@ -841,6 +842,25 @@ def export_token(self, request, pk) -> Response:
             return Response(status=status.HTTP_204_NO_CONTENT)
         return Response(status=status.HTTP_405_METHOD_NOT_ALLOWED)
 
+    @action(methods=["get"], detail=False)
```

Comment:
I believe we have openapi docs for the `/users` routes ([code](https://github.com/grafana/oncall/blob/dev/engine/settings/base.py#L326)). do we need to add anything here to document this new endpoint?

Reply:
good point, added